sim/src/analytics.rs

```diff
@@ -225,17 +219,6 @@ impl Analytics {
             }
         }
 
-        // Lane Speed
-        if let Event::LaneSpeedPercentage(trip_id, lane_id, avg_speed, max_speed) = ev {
```

Comment:
Is this metric just going away?

Reply:
Yeah, I'm not really sure it adds value, and I actually couldn't get it to show up in the UI before this change. If somebody goes very slow along a lane, then that's because they were delayed by the upcoming intersection. So there should be a delay captured and shown there. Doing both is confusing; how do you attribute the problem to the road or intersection?

Reply:
That makes sense. I have no qualms with removing it.